tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/DependencyResolver.scala

```diff
@@ -8,15 +8,15 @@ import JSLibResolveException.Problem
 object DependencyResolver {
 
   type DependencyFilter =
-    Traversable[FlatJSDependency] => Traversable[FlatJSDependency]
+    Iterable[FlatJSDependency] => Iterable[FlatJSDependency]
```

Comment:
That is not an acceptable change, because it breaks a public API.

You need to use a `type` alias in an `Compat` object instead.